package.json

```diff
@@ -1,30 +1,29 @@
 {
   "name": "nemo",
-  "version": "0.4.0",
+  "version": "1.0.4",
   "description": "Abstraction for automation test components",
   "scripts": {
-    "test": "grunt --grep @travis@"
+    "test": "grunt"
   },
   "dependencies": {
-    "selenium-webdriver": "~2.44.0",
-    "xunit-file": "~0.0.4",
+    "async": "~0.2.8",
+    "confit": "^2.0.0",
     "debug": "^2.1.0",
-    "async": "0.1.x",
+    "lodash": "^2.4.0",
     "nconf": "~0.6.7",
-    "lodash": "^2.4.0"
+    "path": "^0.11.14",
+    "selenium-webdriver": "~2.44.0",
```

Comment:
Unfortunately we cannot. 2.45.1 will not launch either Chrome or Firefox on Mac OS. We will have to stay glued to 2.44 until they resolve that issue.